README.md

```diff
@@ -14,7 +14,8 @@ integer elements:
 You don't need to validate arguments (they are always valid).
 
 Examples:
-```
+``
```

Comment:
Do not change readme

Reply:
Very strange, I didn't change anything in README. In my PyCharm correct 
![image](https://user-images.githubusercontent.com/120219525/218324780-90c25b7c-8a16-4e4f-8018-0d32dd2d96bd.png)